Crypt/GPGAbstract.php

```diff
@@ -249,6 +249,7 @@ abstract class Crypt_GPGAbstract
      *                                       will not be executed via destructor.
      * - <kbd>string digest-algo</kbd>     - Sets the message digest algorithm.
      * - <kbd>string cipher-algo</kbd>     - Sets the symmetric cipher.
+     * - <kbd>string compress-algo</kbd>   - Sets the compress algorithm.
```

Comment:
"Compression algorithm". Other than that it's ready to merge. Thanks.

Reply:
I fixed this typo, thank you